### ISSUE TITLE
bug: github.com/m4ll0k/Infoga does not exist anymore which causes celery to fail

### ISSUE THREAD

**Issue by OffS3c** [yogeshojha/rengine#1248]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

celery fails to scan domain

### Expected Behavior

scans should not fail

### Steps To Reproduce

1. start install on vps
2. while installing [rengine](https://github.com/yogeshojha/rengine/tree/master)/[web](https://github.com/yogeshojha/rengine/tree/master/web)
/celery-entrypoint.sh, you will see that `git clone https://github.com/m4ll0k/Infoga /usr/src/github/Infoga` fails because the repo does not exist anymore. Causing celery to fail later.

### Environment

```markdown
- reNgine: 2.0.5
- OS: Ubuntu 22.04
- Python: default
- Docker Engine: latest
- Docker Compose: latest
- Browser: chrome latest
```

### Anything else?

![image](https://github.com/yogeshojha/rengine/assets/27676755/db863b21-c35c-4f73-ab0f-f10ac7aa633a)

**Comment by OffS3c** [comment]
I was also able to port this to m2 pro. but right now it seems getting this bug resolved is more priority. I have extensive cybersec and development experience, would you like us to collaborate together? I'm open for contributions. 😄

**Comment by Talanor** [comment]
Probable duplicate of #1097 fixed in #1099
